build.js

```diff
@@ -131,25 +132,22 @@ const injectHtmlPages = async (metafile, revision) => {
     // find the -index-*.css file in the metafile results
     const cssFile = Object.keys(metafile.outputs).find(file => file.endsWith('.css') && file.includes('ipfs-sw-index'))
 
-    if (!cssFile) {
-      console.error('Could not find the ipfs-sw-index CSS file in the metafile.')
-      return
-    }
-
     const logoFile = Object.keys(metafile.outputs).find(file => file.endsWith('.svg') && file.includes('ipfs-sw-ipfs-logo'))
+    let htmlContent = await fs.readFile(htmlFilePath, 'utf8')
 
-    if (!logoFile) {
-      console.error('Could not find the ipfs-sw-ipfs-logo SVG file in the metafile.')
-      return
+    if (cssFile != null) {
+      const cssTag = `<link rel="stylesheet" href="/${path.basename(cssFile)}">`
+      htmlContent = htmlContent.replace(/<%= CSS_STYLES %>/g, cssTag)
+      console.log(`Injected ${path.basename(cssFile)} into ${htmlFilePath}.`)
     }
 
-    const cssTag = `<link rel="stylesheet" href="/${path.basename(cssFile)}">`
+    if (logoFile != null) {
+      const logoTag = `<img src="/${path.basename(logoFile)}" alt="IPFS Logo">`
+      htmlContent = htmlContent.replace(/<%= IPFS_LOGO_PATH %>/g, logoTag)
+      console.log(`Injected ${path.basename(logoFile)} into ${htmlFilePath}.`)
+    }
```

Comment:
do we want to allow the logo file to not be available?

if we mark it as an entry in our build-config, it should always be available and we will always inject it..